examples/awesome-vault-service/challenger/main.go

```diff
@@ -17,68 +17,91 @@ import (
 )
 
 func main() {
-	logger, err := logging.NewZapLogger(logging.Production)
+	// 0. Create the logger where all the logs will appear
+	logger, err := logging.NewZapLogger(logging.Production) // Change here if want to change logging level
 	if err != nil {
 		println("Failure creating logger")
 		return
 	}
 
+	// 1. Create the config passed to the challenger, also creating the values inside it before
```

Comment:
Let's drop the last part:

```suggestion
	// 1. Create the config passed to the challenger
```